doc/source/user_guide/groupby.rst

```diff
@@ -1195,8 +1199,8 @@ function.
 
 .. note::
 
-   All of the examples in this section can be more reliably, and more efficiently,
-   computed using other pandas functionality.
+   All of the examples in this section can be more reliably, and more efficiently
+   computed using other pandas functionalities.
```

Comment:
`and more efficiently` is a nonrestrictive clause, so it should have a comma after it. I also think functionality should remain singular here - but don't feel certain about this case.

Reply:
I will leave it as it was.